public/app/features/alerting/unified/utils/time.test.ts

```diff
@@ -13,3 +13,41 @@ describe('isValidPrometheusDuration', () => {
     expect(isValidPrometheusDuration(duration)).toBe(false);
   });
 });
+
+describe('parsePrometheusDuration', () => {
+  const tests: Array<[string, number]> = [
```

Comment:
Could just define this once and reuse it across the two tests? Might make it clearer to spot that the input/output are swapped based on the fact that its reused